house/views.py

```diff
@@ -1,21 +1,29 @@
-from django.shortcuts import render
-from house.constants import HOME_PAGE_ROUTE, GLOBAL_PAGE_ROUTE
+from house.constants import HOME_PAGE_ROUTE, HOUSE_PAGE_ROUTE
+from .models import House
+from expenses.models import Expenses
+from django.shortcuts import render, get_object_or_404
 
 
 def home_page(request):
+    # this line is needed till the login of the mine page will create and merge
+    print(House.objects.all().order_by("?").first().house_id)
     return render(request, HOME_PAGE_ROUTE)
 
 
 def global_page(request):
-    return render(request, GLOBAL_PAGE_ROUTE)
+    raise NotImplementedError
 
 
 def house_login(request):
     raise NotImplementedError
 
 
 def house_view(request, house_id):
-    raise NotImplementedError
+    house = get_object_or_404(House, pk=house_id)
```

Comment:
I read about get_object_or_404
nice use 👍 
I think it could be appropriate in the login page as well, I'll try adding it

Reply:
I don't think we should use it in the login, If a user has some errors with the login we need to display him the error and not send him to 404. for example display him "there is no house with **** house_id"
@LiorKGOW